src/RealtimeClient.ts

```diff
@@ -1,22 +1,31 @@
+import type { WebSocket as WSWebSocket } from 'ws'
+
 import {
-  VSN,
   CHANNEL_EVENTS,
-  TRANSPORTS,
-  SOCKET_STATES,
+  CONNECTION_STATE,
+  DEFAULT_HEADERS,
   DEFAULT_TIMEOUT,
+  SOCKET_STATES,
+  TRANSPORTS,
+  VSN,
   WS_CLOSE_NORMAL,
-  DEFAULT_HEADERS,
-  CONNECTION_STATE,
 } from './lib/constants'
-import Timer from './lib/timer'
 import Serializer from './lib/serializer'
+import Timer from './lib/timer'
+
+import { httpEndpointURL } from './lib/transformers'
 import RealtimeChannel from './RealtimeChannel'
 import type { RealtimeChannelOptions } from './RealtimeChannel'
 
-import type { WebSocket as WSWebSocket } from 'ws'
-
 type Fetch = typeof fetch
 
+export type Channel = {
+  name: string
+  inserted_at: string
+  updated_at: string
+  id: number
+}
+
```

Comment:
@filipecabaco It seems like this type isn't being used, but what is it for?

Reply:
will check with @w3b6x9 🤔